Formula/libngspice.rb

```diff
@@ -37,6 +28,14 @@ class Libngspice < Formula
     depends_on "libtool" => :build
   end
 
+  conflicts_with "ngspice", because: "both install initdata scripts"
```

Comment:
Are these initdata scripts identical? If so we should probably just delete them from one formula (probably this one)

Reply:
yeah, the files are identical